server/ServerFSM.cpp

```diff
@@ -1321,6 +1370,11 @@ sc::result WaitingForTurnEndIdle::react(const SaveGameRequest& msg) {
     return transit<WaitingForSaveData>();
 }
 
+sc::result WaitingForTurnEndIdle::react(const Error& msg) {
```

Comment:
How is that relevant to my question that it follows (regarding overriding functions seemingly unnecessarily)?